groups/bdl/bdls/bdls_filesystemutil.cpp

```diff
@@ -1247,7 +1247,7 @@ int FilesystemUtil::visitPaths(
     if (bsl::string::npos != dirName.find_first_of("*?")) {
         bsl::vector<bsl::string> leaves;
         bsl::vector<bsl::string> paths, workingPaths;
-        bsl::string pattern = patternSV;
+        bsl::string pattern(patternSV.data(), patternSV.size());
```

Comment:
```suggestion
        bsl::string pattern(patternSV);
```

Reply:
Ok.  That change is already on master.  Shall we close this PR then?